CuddleTreeDefenseGame/Assets/Scripts/ControlHandler.cs

```diff
@@ -1,16 +1,43 @@
 using UnityEngine;
+using System.Collections;
 
 public class ControlHandler : MonoBehaviour
 {
     [SerializeField] GameObject buildPrefabWithButtonT;
+    private bool isTPressed;
+    private GameObject obj;
```

Comment:
Can be a local variable inside Update(), since its not used anywhere else.

Reply:
Not really, it needs to be used on all updates, if I make it local it will be overriden. If this would have been c++ we could have solved this with static maybe, but not here